packages/j/jabref/package.yml

```diff
@@ -1,34 +1,36 @@
 name       : jabref
-version    : '5.5'
-release    : 4
+version    : '5.15'
+release    : 5
 source     :
-    - https://github.com/JabRef/jabref/archive/refs/tags/v5.5.tar.gz : af0d7c5566aec94ffa8b5912c116956ccf17331e5b46bb5adbd6afeaa85ac230
+    - https://github.com/JabRef/jabref/archive/refs/tags/v5.15.tar.gz : 2bf75d7f96aa1f5c74b675e3e0b4dda703027b3c3d8ede0616dd2218f5a74259
+homepage   : https://jabref.org
 license    : MIT
 component  : office
-summary    : Graphical Java application for managing BibTeX and BibLaTeX (.bib) databases
+summary    : JabRef Bibliography Management 
 description: |
-    Graphical Java application for managing BibTeX and BibLaTeX (.bib) databases
+    JabRef is an open-source, cross-platform citation and reference management tool.
+    JabRef helps you to collect and organize sources, find the paper you need and discover the latest research.
 builddeps  :
+    - binutils
     - gradle
-    - openjdk-17
+    - openjdk-21
 rundeps    :
-    - openjdk-17
+    - openjdk-21
 networking : yes
 environment: |
-    export JAVA_HOME=/usr/lib64/openjdk-17
+    export JAVA_HOME=/usr/lib64/openjdk-21
     export PATH=$JAVA_HOME/bin:$PATH
 
     export GRADLE_USER_HOME=$workdir
 build      : |
-    gradle \
-      --no-daemon \
-      -PprojVersionInfo=$version \
-      -PprojVersion=$version \
-      assemble
+    gradle assemble --no-daemon --no-build-cache -PprojVersionInfo=$version -PprojVersion=$version 
+    gradle jlink
```

Comment:
I don't see a point in using `gradle jlink`. That builds a distribution for jabref which includes the openjdk (so you can use it without java installed), but that's not really desirable for us since that openjdk is built from whatever the openjdk version was at the build time but if we update openjdk then jabref will still be using an old openjdk until it's rebuilt.

Reply:
Yes.  I agree.  I have a new build that does not use that, but I have not been able update the PR yet due to some other issues.   I will try to that soon for your review.